wifiphisher/common/interfaces.py

```diff
@@ -243,6 +243,36 @@ def up_ifaces(self, ifaces):
         for i in ifaces:
             card = pyw.getcard(i.get_name())
             pyw.up(card)
+    
+    def _generate_random_address(self):
+        """
+        Make and return the randomized MAC address
+
+        :param self: A NetworkManager object
+        :type self: NetworkManager
+        :return: A MAC address
+        :rtype str
+        """
+
+        mac_addr = "02:00:00:{:02x}:{:02x}:{:02x}".format(random.randint(0, 255),\
```

Comment:
When randomize MAC address, the OUI must be the exist one. If we just called  `random.randint(0, 255)` for the OUI, it may set the MAC address fail, since the OUI doesn't exist. Originally I thought we may randomly pick one OUI from the OUI fie wifiphisher has; however it may be too much for just getting a randomized address?